### ISSUE TITLE
MaxListenersExceededWarning. Possible EventEmitter memory leak detected

### ISSUE THREAD

**Issue by defnull** [bigbluebutton/bbb-webrtc-sfu#125]
Some other admin (not me) got the following error:

    bbb-webrtc-sfu[2463686]: (node:2463686) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 DisconnectAllClientsSysMsgdd5916b92e8b69db9f8820f1fab7e4700bf6fb8d-1683023698009 listeners added to [BigBlueButtonGW]. Use emitter.setMaxListeners() to increase limit
    bbb-webrtc-sfu[2463686]: (node:2463686) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 MCS_DISCONNECTED listeners added to [MCSAPIWrapper]. Use emitter.setMaxListeners() to increase limit

`EventEmitter` will only allow 10 listeners per event by default, unless told otherwise. The first line is almost certainly triggered by https://github.com/bigbluebutton/bbb-webrtc-sfu/blob/240a585b5fac36598766d4e29bcf235c4102c786/lib/video/video.js#L131. I'm not sure if this is a leak (event not properly de-registered in certain scenarios) or if there are actually more than 10 valid instance of `Video` per meeting at the same time (in which case the limit should be increased?), or if it is something else.

**Comment by prlanzarin** [comment]
Hello @defnull,

> . I'm not sure if this is a leak (event not properly de-registered in certain scenarios) or if there are actually more than 10 valid instance of Video per meeting at the same time (in which case the limit should be increased?), or if it is something else.

1. It's not a leak - events are properly de-registered last time I reviewed that code.
2. Yes, there's more than one `Video` instances per meeting (it's 1 per stream)
3. I did not increase the limits - intentionally. The warning is benign, there's [no hard limit being hit there](https://nodejs.org/api/events.html#eventsdefaultmaxlisteners) nor should there be any leak. I left it untouched as a reminder to myself: the way those events handlers are set up is not ideal (design wise), so the logs are a frequent reminder that I have to eventually refactor that stuff.
